wiki/Ranking_Criteria/en.md

```diff
@@ -106,7 +106,7 @@ Changes to the rules and guidelines in this document are proposed and discussed
 
 #### Romanisation
 
-- **An artist's name is to be romanised in the order it is printed in the unicode field, unless an alternative order for the romanised name is provided by the artist.**
+- **When romanising an artist's name yourself and no official romanisation is available, it must be romanised in the order it is printed in the unicode field.**
```

Comment:
```suggestion
- **When romanising an artist's name yourself (ie. no official romanisation is available), it must be romanised in the order it is printed in the unicode field.**
```

I think you wouldn't be romanising yourself if there was an official one, correct?

Reply:
Implying that you shouldn't romanise the artist name yourself if there is an official one available contradicts with the allowance, since it states that picking the official romanisation or translation is optional. Again, any preference towards which option to pick is out of scope for this proposal as [Noffy stated](https://github.com/ppy/osu-wiki/pull/6079#discussion_r706344648).

Reply:
so you are allowed to romanise differently to the artist's preference?? that seems very counter-intuitive.

we should just fix that with the proposed wording change here. @Ephemeralis second opinion on this weirdness please.

Reply:
I'm not ephemeral but opened PR https://github.com/ppy/osu-wiki/pull/6149 to cover this instead. I think this current PR should be good to go as is once that is merged.